app/UI/message-list.tsx

```diff
@@ -41,6 +42,7 @@ interface Props {
         relayRecordGetter: RelayRecordGetter;
         getEventByID: func_GetEventByID;
     };
+    onReplyToEventIDChange?: (eventID?: string | NoteID) => void;
```

Comment:
If we can get the `NostrEvent` at thie level, passing an `NostrEvent` might be better. `string | NoteID` might be too small for this interface. Up to you.